developers.csv

```diff
@@ -81,7 +81,7 @@ Philip Jenvey,pjenvey,2009-05-07,2020-11-26,For Jython compatibility
 Michael Foord,voidspace,2009-04-01,,For IronPython compatibility
 R\. David Murray,bitdancer,2009-03-30,,
 Chris Withers,cjw296,2009-03-08,,
-Tarek Ziadé,,2008-12-21,2017-02-10,For distutils module; did not make GitHub transition
+Tarek Ziadé,tarekziade,2008-12-21,2017-02-10,For distutils module

```

Comment:
This file is generated from https://github.com/python/voters/ , so please update there and then update the file via the GitHub Action which creates the CSV file.

Reply:
I created https://github.com/python/voters/pull/64.